src/runtime/program/vote/instruction.zig

```diff
@@ -0,0 +1,43 @@
+const std = @import("std");
+const sig = @import("../../../sig.zig");
+
+const Pubkey = sig.core.Pubkey;
+
+pub const IntializeAccount = struct {
+    node_pubkey: Pubkey,
+    /// The vote authority keypair signs vote transactions. Can be the same as the identity account.
+    authorized_voter: Pubkey,
+    /// The authorized withdrawer keypair is used to withdraw funds from a vote account,
+    /// including validator rewards. Only this keypair can access the funds.
+    authorized_withdrawer: Pubkey,
+    /// Commission is the percentage of network rewards kept by the validator.
+    /// The rest is distributed to delegators based on their stake weight.
+    commission: u8,
+
+    const AccountIndex = enum(u8) {
+        /// `[WRITE]` Uninitialized vote account
+        account = 0,
+        /// `[]` Rent sysvar
+        rent_sysvar = 1,
+        /// `[]` Clock sysvar
+        clock_sysvar = 2,
+        /// `[SIGNER]` New validator identity (node_pubkey)
+        signer = 3,
+    };
+
+    pub fn accountIndex(index: AccountIndex) u8 {
```

Comment:
would rather not have this helper method -- just use `intFromEnum` directly

Reply:
Removed in https://github.com/Syndica/sig/commit/edaab81573c577d564d6a850fd300d6b0ad83b86